claims_errors.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2021-2022 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package psatoken
+
+import "errors"
+
+var (
+	ErrClaimUndefined        = errors.New("undefined claim")
+	ErrOptionalClaimMissing  = errors.New("missing optional claim")
```

Comment:
What is the rationale behind tagging a Missing Optional Claim as error? I saw in other file, you check for errors and then filter out specific error for ErrOptionalClaimMissing ?

Reply:
this error exists to provide an explicit signal (for a caller that uses the interface to access the claims' values) that the requested claim is not set but that's allowed by the specific profile.